sdk/storage/Azure.Storage.DataMovement/src/Shared/StorageResourceContainerInternal.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Threading;
+
+namespace Azure.Storage.DataMovement
+{
+    internal abstract class StorageResourceContainerInternal : StorageResourceContainer
```

Comment:
NIT - comment above class just as an FYI of what this is doing.

Reply:
LMAO i forgot it on my own PR after asking you